Specs/DateTime/French/SetExtractor.json

```diff
@@ -33,13 +33,13 @@
     ]
   },
   {
-    "Input": "Je vais partir 9 lundis",
```

Comment:
Does "Je vais partir à 9 lundis" work?

Reply:
"9 lundis" can be extracted but without parse result the same as the previous version.

Reply:
OK. Please add this case in the spec.